migrations/2_deploy_contracts.js

```diff
@@ -2,10 +2,12 @@ var RelayHub = artifacts.require('./RelayHub.sol')
 var StakeManager = artifacts.require('./StakeManager.sol')
 var Penalizer = artifacts.require('./Penalizer.sol')
 var SampleRecipient = artifacts.require('./test/TestRecipient.sol')
+var Eip712Forwarder = artifacts.require('Eip712Forwarder')
 
 module.exports = async function (deployer) {
   await deployer.deploy(StakeManager)
   await deployer.deploy(Penalizer)
   await deployer.deploy(RelayHub, StakeManager.address, Penalizer.address)
-  await deployer.deploy(SampleRecipient)
+  await deployer.deploy(Eip712Forwarder)
+  await deployer.deploy(SampleRecipient, Eip712Forwarder.address)
```

Comment:
its just a refactor of SampleRecipient, which now receives a forwrader instead of creating one inside the constructor (which is a bad habbit anyway)
yes, we should remove them - and have some proper deployment utility (e.g. buidler?!?!)